spec/dummy/db/schema.rb

```diff
@@ -232,6 +232,9 @@
     t.string "test_extension"
     t.text "test_template"
     t.boolean "feedback"
+    t.boolean "multifile", default: false
+    t.boolean "layout_shows_loading_content", default: false
```

Comment:
You'll need to rebase this against branch `https://github.com/mumuki/mumuki-laboratory/tree/update-gem-versions`